### PR TITLE
chore: rename `install tls` command to `tls install` to be compliant with new naming convention

### PR DESCRIPTION
**Pull request by loresuso**
This allow to be compliant with the naming we have chosen for this repo and the other commands.

Signed-off-by: Lorenzo Susini <susinilorenzo1@gmail.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

/kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

> /kind flaky-test

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area library

/area cli

> /area tests

> /area examples

**What this PR does / why we need it**:
The naming convention for command is object and then verb, i.e `artifact install`.
To respect this, change `install tls` with `tls install`.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: cmd/tls.go [hunk 1]
```diff
@@ -18,17 +18,17 @@ import (
...
+// NewTLSCmd return the tls command.
+func NewTLSCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:                   "install",
+		Use:                   "tls",
 		TraverseChildren:      true,
 		DisableFlagsInUseLine: true,
-		Short:                 "Install a component with falcoctl",
-		Long:                  `Install a component with falcoctl`,
+		Short:                 "Handle certificates with falcoctl",
```
Comment (loresuso) [inline cmd/tls.go:27]: Is this a right choice for the description? Can we find better?

**Comment by FedeDP** [comment]
> To respect this, change install tls with tls install.

Just out of curiosity: isn't `tls` more like flag of the  `artifact install` command?

**Comment by loresuso** [comment]
> > To respect this, change install tls with tls install.
> 
> Just out of curiosity: isn't `tls` more like flag of the `artifact install` command?

hey @FedeDP, more than a flag `tls` could be a subcommand. I think this is a matter of taste, but since we have chosen the pattern `NOUN` `VERB` for the other commands, I now found having `install tls` a bit misleading, especially because of the `artifact install` command

**Comment by FedeDP** [comment]
> but since we have chosen the pattern NOUN VERB for the other commands

Makes sense; but are we really installing a `tls` ? :)

File: cmd/tls.go (hunk 1 above)
Comment (leogr) [reply cmd/tls.go:27]: My proposal is: `Generate and install TLS material for Falco`

**Comment by leogr** [comment]
Hey @FedeDP @loresuso 

Yes, it actually installs the TLS material (in a sense indicated by the description: `This command is a convenience to not only generate the TLS material, but also drop it off on the local filesystem.`).

@loresuso this PR will break that blog post anyway, so I agree that the less we change, the better, but we still have to indicate this as a breaking change and maybe update that blog post too.


PS
left only one suggestion, otherwise SGTM

**Review by leogr — APPROVED** [review]
LGTM

**Pull request merged**